netlify/functions/resources.ts

```diff
@@ -68,13 +68,45 @@ const resourcesHandler: Handler = async (event, context) => {
       body: tableToCsv(tableData),
     };
   } else if (resourceFilename === "findings.csv") {
-    const rawData = await readFile("./_data/findings/findings.csv", "utf8");
+    const findingsData = await getApiFindingsData();
+    let tableData: string[][] = [
+      [
+        "contest",
+        "handle",
+        "finding",
+        "risk",
+        "score",
+        "pie",
+        "split",
+        "slice",
+        "award",
+        "awardCoin",
+        "awardUSD",
+      ],
+      //contest,handle,finding,risk,score,pie,split,slice,award,awardCoin,awardUSD
+      ...findingsData.map((finding) => {
+        return [
+          "" + finding.contest,
+          finding.handle,
+          finding.finding,
+          finding.risk,
+          "" + finding.score,
+          "" + finding.pie,
+          "" + finding.split,
+          "" + finding.slice,
+          "" + finding.award,
+          finding.awardCoin,
+          "" + finding.awardUSD,
```

Comment:
I think after https://github.com/code-423n4/api.code4rena.com/pull/534#discussion_r1169384518 most of these should be good with the default rounding but let's tweak the final column to mostly match current formatting:

```suggestion
          "" + finding.pie,
          finding.split.toPrecision(6),
          "" + finding.slice,
          "" + finding.award,
          finding.awardCoin,
          finding.awardUSD.toPrecision(2),
```

This will still change some of the current `0` cells to become `0.00` but I think that's reasonable and shouldn't cause any backwards-compatibility trouble!